src/modules/whats-new/components/whats-new-modal.tsx

```diff
@@ -58,6 +58,14 @@ export default function WhatsNewModal( { showModal, onClose }: WhatsNewModalProp
 				'Discover the latest updates in Studio! Explore exciting new features designed to enhance your experience.'
 			),
 		},
+		{
+			image: studioIllustration,
+			title: __( 'Introducing Studio by WordPress.com CLI' ),
```

Comment:
Should we flip the wording a bit? Or maybe we could skip WordPress.com here?

```suggestion
			title: __( 'Introducing Studio CLI by WordPress.com' ),
```

Reply:
I think it should be consistent with the CLI.
Should we adjust the wording in both places?

![image](https://github.com/user-attachments/assets/38f2ba4c-412b-4581-8a20-def930308f2b)

Reply:
In this context, I agree with Wojtek that we can skip "by WordPress.com"